ocean_lib/models/erc721_token.py

```diff
@@ -195,6 +195,9 @@ def safe_transfer_from(
     def withdraw(self, from_wallet: Wallet):
         return self.send_transaction("withdrawETH", (), from_wallet)
 
+    def token_name(self) -> str:
```

Comment:
I encountered this issue too. The .name PROPERTY is the Contract name, and it would be confusing for the name() FUNCTION  to return the token name. I agree with this solution.